tests/vasp/flows/test_defect.py

```diff
@@ -0,0 +1,127 @@
+import numpy as np
+
+from atomate2.vasp.flows.defect import ConfigurationCoordinateMaker
```

Comment:
I think if you move these imports inside the tests themselves it should fix the psuedopotential issue.

Kind of annoying, I don't know how to fix it permanently.